ql/src/test/org/apache/hadoop/hive/ql/txn/compactor/CompactorTest.java

```diff
@@ -271,6 +273,11 @@ protected void addLegacyFile(Table t, Partition p, int numRecords) throws Except
     addFile(t, p, 0, 0, numRecords, FileType.LEGACY, 2, true);
   }
 
+  protected void addDeltaFile(Table t, Partition p, long minTxn, long maxTxn, int numRecords,
+      long visibilityId) throws Exception {
+    addFile(t, p, minTxn, maxTxn, numRecords, FileType.DELTA, 2, true, visibilityId);
```

Comment:
this change has 0 effect, as it visibilityId is not used in delta file creation. Should be 
````
case DELTA: filename = AcidUtils.addVisibilitySuffix(
    makeDeltaDirName(minTxn, maxTxn),
    visibilityId); 
````